IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_RailwayTensioningSectionGeneral/DocPropertySet.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RailwayTensioningSectionGeneral" UniqueId="080715c0-01c0-25a8-6f15-d9cc7ab1f55e" ApplicableType="IfcSpatialZone/RESERVATION" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
what is the reasoning for this being seperate to Pset_SpatialZoneTypeReservationRailwayTensioningSection?

Reply:
I moved this Pset out of this pull request. I consider it as a backlog issue